amundsen_application/static/css/_bootstrap-custom.scss

```diff
@@ -12,7 +12,7 @@ $icon-font-path: '/static/fonts/bootstrap/';
 
 // Core CSS
 @import '~bootstrap-sass/assets/stylesheets/bootstrap/scaffolding';
-@import '~bootstrap-sass/assets/stylesheets/bootstrap/code';
+// @import '~bootstrap-sass/assets/stylesheets/bootstrap/code';
```

Comment:
I left this line here as a reference, but happy to remove it overall

Reply:
The styles on this file were colliding with the styles we apply with our theme.

I didn't find any other place where we are using a <code> or <pre> tag, so I thought it was safe!

Reply:
Thanks, I'd say you can remove of leave comment inline with the context of why it's commented out. I lean towards preferring the comment with context instead of having this nuance lost to tribal knowledge and have the issue you ran into re-introduced in the future

Reply:
Just added it!